sys/stat_windows.go

```diff
@@ -0,0 +1,74 @@
+package sys
+
+import (
+	"os"
+)
+
+//NOTE: taken from amd64 Linux
+type Timespec struct {
+	Sec  int64
+	Nsec int64
+}
+
+type Stat_t struct {
+	Dev     uint64
+	Ino     uint64
+	Nlink   uint64
+	Mode    uint32
+	Uid     uint32
+	Gid     uint32
+	Rdev    uint64
+	Size    int64
+	Blksize int64
+	Blocks  int64
+	Atim    Timespec
+	Mtim    Timespec
+	Ctim    Timespec
+}
+
+// InfoToStat takes a platform native FileInfo and converts it into a 9P2000.L compatible Stat_t
+func InfoToStat(fi os.FileInfo) *Stat_t {
+	return &Stat_t{
+		Size: fi.Size(),
+		Mode: uint32(modeFromOS(fi.Mode())),
+		Mtim: Timespec{
+			Sec:  fi.ModTime().Unix(),
+			Nsec: fi.ModTime().UnixNano(),
+		},
+	}
+
+}
+
+//TODO: copied from pkg p9
+// we should probably migrate the OS methods from p9 into sys
```

Comment:
hmmm. probably, yeah.

I'll probably end up making a copy of gVisor's pkg/abi/linux for this at some point.

Reply:
sys sgtm for now, though